data/json/items/armor.json

```diff
@@ -336,6 +336,28 @@
     "material_thickness": 3,
     "flags": [ "BELTED", "BLOCK_WHILE_WORN" ]
   },
+  {
+    "id": "armguard_scrap",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "pair of scrap arm guards",
+    "description": "A pair of arm guards made from scraps of metal secured by simple strings; the loose collection of plates provides decent but not the most convenient protection.",
+    "weight": 3064,
+    "volume": 19,
+    "price": 20000,
+    "to_hit": 1,
+    "bashing": 1,
+    "cutting": 1,
+    "material": [ "steel", "iron" ],
```

Comment:
I wonder if we need to add a `cotton` material here since cordage is required to craft.

Reply:
Well... the Cordage isn't sufficient to be considered cotton in material as it's just something that ties the scrap together, not something that covers the skin from the scrap